TTS/__init__.py

```diff
@@ -31,7 +32,7 @@
     # Bark
     torch.serialization.add_safe_globals(
         [
-            np._core.multiarray.scalar,
+            (_np_scalar, "numpy.core.multiarray.scalar"),
```

Comment:
Strings don't work here. You can add the following line taken from the HuggingFace transformers repo just after the `# Bark` comment (needs `from packaging import version`): `np_core = np._core if version.parse(np.__version__) >= version.parse("2.0.0") else np.core` Then:

```suggestion
            np_core.multiarray.scalar,
```